release_notes/topics/18_0_0.adoc

```diff
@@ -1,5 +1,22 @@
 = Highlights
 
+== New Operator tech-preview
+
+With this release, we're introducing a brand new {project_operator} as a tech-preview. Apart from being rewritten from
+scratch, the main user-facing change from the legacy Operator is the used {project_name} distribution – the new Operator
+uses the Quarkus distribution of {project_name}. With that, there are a few breaking changes. For details, incl.
+installation and migration instructions, see the https://www.keycloak.org/guides[Operator related guides].
+
+The link:{operatorRepo_link}[legacy Operator] will receive updates until June 2022 when the {project_name} WildFly
+distribution reaches EOL.
+
+=== OperatorHub versioning scheme
+To avoid version conflicts with the legacy Operator, the 18.0.0 version of the new Operator is released as version
+`20.0.0-alpha.1` on OperatorHub. The legacy Operator versioning scheme remains the same, i.e. it is released as 18.0.0.
```

Comment:
Just a sidenote: This _could_ slightly impact the overall usage of the new operator btw. I know many companies where compliance does not allow you to  install anything tagged _alpha_, _beta_ or sth. And I know from myself I either delay the usage than filing multiple compliance reports to get the use of a "production ready operator thats only tagged wrong for technical reasons" through the hells of bureaucracy. Can't come up with a better solution myself, though. ;)

Reply:
That's a good point, but I think we don't want to hide too much the fact that it's a tech-preview. After all, the new operator will live in `candidate` channel for now.

Reply:
I have to add that versioning stuff with `alpha` and `beta` is a pretty common practice on Kubernetes land.